src/MediaRental/DatabaseSupport.java

```diff
@@ -1,5 +1,6 @@
 package MediaRental;
 
+import java.lang.reflect.Array;
```

Comment:
I'm not actually using it. Whenever I type "Array" and try to auto-complete to ArrayList, my IDE automatically adds this import statement. It can be safely removed. I am not using it.